### DISCUSSION TITLE
Add Fields To Transactions Table

### DISCUSSION THREAD

**Discussion by sadiqumar18** [bavix/laravel-wallet#968] [Q&A]
Hi,

I'm working on a project where we need to categorize transactions more specifically than what the current `type` field allows. We've been using the `meta` JSON field for this, we're concerned about query performance when filtering by these categories.

Would it be possible to add an option in the package to allow users to define additional fields for the transactions table? For example, we'd like to add a `transaction_category` field.

I understand this would be a significant change, so I wanted to check if this is something you'd consider.

Thanks for your time and for maintaining this great package!

**Comment by rez1dent3** [comment]
Hello. The package allows you to easily add and use new fields that you need:
https://github.com/bavix/laravel-wallet/issues/845
https://github.com/bavix/laravel-wallet/issues/201#issuecomment-1101062835

If you are experiencing problems, please describe them in more detail and I will try to help you.
